src/runtime/id.ts

```diff
@@ -104,3 +116,15 @@ export class Id {
     return true;
   }
 }
+
+export class ArcId extends Id {
+  /** Creates a new Arc ID. Use IdGenerator to create new IDs instead. */
+  static _createArcIdInternal(root: string, name: string): ArcId {
+    return new ArcId(root, [name]);
+  }
+
+  /** Creates a new Arc ID with the given name. For convenience in unit testing only; otherwise use IdGenerator to create new IDs instead. */
+  static newArcIdForTest(id: string): ArcId {
```

Comment:
WDYT about renaming to `newForTest`? `ArcId.newForTest` conveys the same information as `ArcId.newArcIdForTest`, but is shorter, which is nice.

Reply:
Done!